docs/source/commandline/commandline-all.rst

```diff
@@ -695,11 +710,13 @@ Core Video Options
 |	Using '**bgfx**' specifies the new hardware accelerated renderer.
```

Comment:
`bgfx` and `none` are not Windows-specific.  Probably better to have `bgfx`, `opengl` and `none` in a general group first, followed by the OSD- and platform-specific ones in their own groups.

Reply:
Fixed on my side.